### DISCUSSION TITLE
Being able to deprecate components

### DISCUSSION THREAD

**Discussion by peauc** [sveltejs/kit#12165] [Q&A] [answered]
Hello, I am a member of a sveltekit project with a lot of components and we are looking to deprecate some of them.
Is that possible to tag the components so that when a team member imports it, it gets marked a deprecated by the LSP ?

**Comment by dummdidumm** [comment, marked as answer]
This is possible using the [`@component` html comment](https://svelte.dev/docs/faq#how-do-i-document-my-components):
```svelte
<!--
    @component
    @deprecated bla bla bla
-->
<script>
  ..
</script>
..
```

**Reply by peauc** [reply]
Thank you very much. I was looking exactly for that.
